### PR TITLE
add some missing doc strings

### PR DESCRIPTION
**Pull request by tthsqe12**

### CODE REVIEW

**Comment by thofma** [comment]
I don't think we want to add any of these docstrings.

**Comment by tthsqe12** [comment]
the doc page was requesting the. I removed them all now.

File: src/poly/poly.jl [hunk 1]
```diff
@@ -87,6 +87,12 @@ function isquotient_ring(R::PolyRing)
    GC.@preserve R return Bool(Singular.libSingular.rIsQuotientRing(R.ptr))
 end
 
+@doc Markdown.doc"""
+    characteristic(R::PolyRing)
+
+Return the characteristic of $R$. This function is currently unreliable if the
+characteristic does not fit into a `Cint`.
+"""
```
Comment (thofma) [inline src/poly/poly.jl:95]:
```suggestion
```

Comment (tthsqe12) [reply src/poly/poly.jl:95]: This one too? Now its unreliability is not documented...

**Pull request merged**